src/components/Facets.tsx

```diff
@@ -118,9 +168,31 @@ export function getFacetTypeFromReactElementType(elementType: string) {
   switch (elementType) {
     case NumericalFacet.name.toString():
       return FacetType.NUMERICAL;
+    case HierarchicalFacet.name.toString():
+      return FacetType.HIERARCHICAL;
     case StandardFacet.name.toString():
       // fall through
     default:
       return FacetType.STANDARD;
   }
 }
+
+/**
+ * Returns the type of the facet based on facet.
+ * @param facet - {@link DisplayableFacet}
+ * @param delimiter - string
+ * @returns {@link FacetType}
+ *
+ * @internal
+ */
+export function getFacetTypeFromFacetAndDelimiter(facet: DisplayableFacet, delimiter?: string) {
+  if (isHierarchicalFacet(facet, delimiter)) {
```

Comment:
I think this `isHierarchicalFacet` check will cause any standard facet which contains the delimiter to render as a hierarchical facet. Is that correct? I think that could cause issues if a standard facet just happens to contain the delimiter. It might be better to not try to determine if a facet is `isHierarchicalFacet` and only render them if a `HierarchicalFacet` is a child of `Facets`

Reply:
I was thinking about it and figured they can always use <StandardFacet/> to override. Started a thread with Baigel to confirm what works best for him

Reply:
Added hierarchicalFieldIds prop to address this